### PR TITLE
DLReports UI

### PR DESCRIPTION
**Pull request by johnbaldwin**
This PR adds downloadable reports to Figures

### CODE REVIEW

**Comment by bryanlandia** [comment]
@johnbaldwin I cherrypicked one commit from #124 since we actually are using this branch in production on UW Hawthorn to get CSV reports.  I'm not sure what the plans are for merging this branch.

File: figures/pipeline/course_daily_metrics.py [hunk 1]
```diff
@@ -113,6 +114,8 @@ def get_average_progress(course_id, date_for, course_enrollments):
     if len(progress):
         progress_percent = [rec['progress_percent'] for rec in progress]
         average_progress = float(sum(progress_percent)) / float(len(progress_percent))
+        average_progress = float(Decimal(average_progress).quantize(Decimal('.00')))
+
```
Comment (johnbaldwin) [inline figures/pipeline/course_daily_metrics.py]: This is already in Figures `master`

**Comment by johnbaldwin** [comment]
I'm redoing this PR with only the changes in frontend/src as

1. We are no longer keeping compiled frontend assets in `master`
2. the pipeline fix that @bryanlandia did is now in `master`

**Comment by johnbaldwin** [comment]
And I'm going to have to manually pick updates from frontent/src as at least some commits have both compiled assets and frontend/src updates. This helps make the commit history clean

**Comment by johnbaldwin** [comment]
This PR is superceded by PR #146

**Pull request closed without merging**